TopicsImplementations/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,11 @@
+eclipse.preferences.version=1
```

Comment:
Add .settings/ in the .gitignore file, as it is something created by the eclipse.